File: Algo/hr_implement/2/reverse_beautiful_days.js
function beautifulDays(i, j, k) {
    let count =0;
    for(let x = i;x<=j;x++){
        let reverse_number = x.toString().split('').reverse();
        reverse_number = parseInt(reverse_number.join(''));
        
        if((Math.abs(x-reverse_number)) % k ==0) {
            count++;
        }
    }
    
    return count;
}

console.log(beautifulDays(20,23,6));

File: Algo/hr_implement/2/reverse_beautiful_days.py
"""
Given a range of numbered days [i..j], and a number k, determine the number of days in the range that are beautiful.
Beautiful numbers are defined as numbers where i - reverse(i) is evenly divisible by k. 
"""

def beautifulDays(i, j, k):
	count = 0 
	for x in range(i,j+1):
		reverse_number = int(''.join(reversed(str(x))))
		if (abs(x - reverse_number)) % k == 0:
			count +=1

	return count

if __name__ == "__main__":
	print beautifulDays(20,23,6)
